testcafe/runner-drive.js

```diff
@@ -17,14 +17,17 @@ async function runRunner() {
   const response = await runner
     .src([
       //Init data : Unzip archive with files to upload
-      'testcafe/tests/helpers/init-data.js',
+      'testcafe/tests/drive/init-data.js',
       //Tests !
       'testcafe/tests/drive/classification_scenario.js',
       'testcafe/tests/drive/navigation.js',
       'testcafe/tests/drive/folder_sharing_scenario.js',
       'testcafe/tests/drive/file_sharing_scenario.js',
       'testcafe/tests/drive/viewer-feature.js',
-      'testcafe/tests/drive/public-viewer-feature.js'
+      'testcafe/tests/drive/public-viewer-feature.js',
+      //cleanup
+      //Init data : Unzip archive with files to upload
```

Comment:
I'm confused by this comment, what does it mean?

Reply:
It means I copy/paste too quickly! :)